src/control/cmd/daos/filesystem.go

```diff
@@ -69,73 +67,217 @@ func (cmd *fsCopyCmd) Execute(_ []string) error {
 type fsAttrCmd struct {
 	existingContainerCmd
 
-	ChunkSize   ChunkSizeFlag `long:"chunk-size" short:"z" description:"container chunk size"`
```

Comment:
not quite sure i read this right.. chunk-size and oclass options are removed?

Reply:
For the get-attr and reset-* commands, I took those flags out, since they weren't actually used by the command. They are still defined for the set-attr command.